checkbox-provider-ce-oem/bin/cpufreq_governors.py

```diff
@@ -774,6 +817,10 @@ def main():
         info.print_policies_list()
         return 0
 
+    if args.probe_module:
+        status = probe_governor_module(info.get_supported_governors())
```

Comment:
I think you could use `info.governors` directly instead of calling `get_supported_governors()` again.